.github/workflows/ci.yml

```diff
@@ -72,6 +72,14 @@ env:
   ALPAKA_ACC_GPU_HIP_ONLY_MODE: OFF
 
 jobs:
+  clang-format:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: DoozyX/clang-format-lint-action@v0.11
+      with:
+        exclude: './thirdParty'
```

Comment:
We should not need this exclude anymore after #781 has been merged.

Reply:
Right!